pytest/tests/shardnet/restake.py

```diff
@@ -0,0 +1,70 @@
+import argparse
+import shlex
+import random
+import sys
+from rc import pmap
+import pathlib
+
+sys.path.append(str(pathlib.Path(__file__).resolve().parents[2] / 'lib'))
```

Comment:
not related to this PR -- @mina86 - is there a better solution for this? could we set some 'env' instead?